__tests__/project.test.js

```diff
@@ -131,3 +129,40 @@ describe("CORE: GET /api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("gets all comments on an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body, "<-- body");
+        body.forEach((element) => {
+          expect(element).toHaveProperty("comment_id");
+          expect(element).toHaveProperty("votes");
+          expect(element).toHaveProperty("created_at");
+          expect(element).toHaveProperty("author");
+          expect(element).toHaveProperty("created_at");
+          expect(element).toHaveProperty("votes");
+          expect(element).toHaveProperty("body");
+          expect(element).toHaveProperty("article_id");
+        });
+      });
```

Comment:
This is a good start, but body should not be the array itself. Body should be an object with a key like comments which holds the array of comments. Element is also not a good name for what is each individual comment.

Before the forEach we should assert the length of the array to ensure it isnt empty. This is because if it was an empty array the forEach would be skipped and this test would pass which a false positive

We also are not really checking if these comments are in fact linked to the article id we are looking at. To do this easly instead of asserting that the article_id exists we should expect it to be the same value as the article_id in the url.